support/packages/github-to-jira-synchronization-tool/src/utils/runSequentially.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+let promise = Promise.resolve();
+
+module.exports = function runSequentially(fn) {
+	promise = promise.then(async () => {
```

Comment:
This may look a bit rudimentary but it does fit my needs here

Reply:
We may optimise this a bit and only process related webhooks sequentially but since the amount of webhooks is not too big I think it is not worth the effort

Reply:
I had something similar in the bundler for file ops and after some tests found out that it was better to do things sequentially than in parallel :sweat_smile: 

In general, sequential leads to more throughput and less interactivity. But given that this is a daemon/service, it may even make sense, who knows without testing :shrug: 

TL;DR: if it works, leave it like this. There's time to optimize it.